apps/condo/domains/contact/schema/Contact.js

```diff
@@ -107,7 +108,7 @@ const Contact = new GQLListSchema('Contact', {
                 if (contact) {
                     return addValidationError('Cannot create contact, because another contact with the same provided set of "property", "unitName", "name", "phone"')
                 }
-            } else if (operation === 'update') {
+            } else if (operation === 'update' && isNil(deletedAt)) {
```

Comment:
So, if the object is deleted, but we did not have the value of the `deletedAt` field (i know its an extreme case) we will allow to update object, while we shouldnt